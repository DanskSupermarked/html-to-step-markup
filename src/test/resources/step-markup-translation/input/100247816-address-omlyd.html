<p>Vores Egen Vinkælder i føtex byder velkommen til en vinsmagning med 16 leverandører  med vine fra hele verden. Der vil både være mulighed for at smage de gode sortimentsvine, men også vine som ikke normalt er tilgængelige i vores butikker. Vinmagerne står klar til, at fortælle om de enkelte vine og guide dig rundt i et paradis for vinelskere.</p> <p>Du kan møde GMDF (Louis Bernard), Montgras, Treasury Wine Estate, Cantina di Soave, Dom&auml;ne Wachau, Undurraga, Trapiche, Delicato, KWV, Cave Vinicole de Ribeauville, J.M. Fonseca, Baron de Ley, Bodegas Castano, Jacquart, Bersano,  og Accolade Wines.</p> <p><strong>Børsen, København, 2. november 2016 kl 19.00-21.30</strong></p> <address>Børsen<br /> Slotsholmsgade 1<br /> 1216 København</address> <p>OBS! På grund af tekniske vanskeligheder, kan brugere med Hotmail e-mail adresser ikke benyttes af dette produkt. Benyt venligst en anden e-mail adresse.</p>
