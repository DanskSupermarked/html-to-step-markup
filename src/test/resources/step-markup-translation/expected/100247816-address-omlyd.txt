Vores Egen Vinkælder i føtex byder velkommen til en vinsmagning med 16 leverandører med vine fra hele verden. Der vil både være mulighed for at smage de gode sortimentsvine, men også vine som ikke normalt er tilgængelige i vores butikker. Vinmagerne står klar til, at fortælle om de enkelte vine og guide dig rundt i et paradis for vinelskere.<return/>
<return/>
Du kan møde GMDF (Louis Bernard), Montgras, Treasury Wine Estate, Cantina di Soave, Domäne Wachau, Undurraga, Trapiche, Delicato, KWV, Cave Vinicole de Ribeauville, J.M. Fonseca, Baron de Ley, Bodegas Castano, Jacquart, Bersano, og Accolade Wines.<return/>
<return/>
<bold>Børsen, København, 2. november 2016 kl 19.00-21.30</bold><return/>
<return/>
Børsen<return/>
Slotsholmsgade 1<return/>
1216 København<return/>
<return/>
OBS! På grund af tekniske vanskeligheder, kan brugere med Hotmail e-mail adresser ikke benyttes af dette produkt. Benyt venligst en anden e-mail adresse.
